src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -2891,6 +2898,51 @@ void InitZoneAirSetPoints(EnergyPlusData &state)
                                 OutputProcessor::SOVTimeStepType::System,
                                 OutputProcessor::SOVStoreType::Average,
                                 Zone(Loop).Name);
+            if (state.dataHeatBal->DoLatentSizing) {
+                SetupOutputVariable(state,
+                                    "Zone Air System Latent Heating Energy",
```

Comment:
These new output variables are useful with or without latent sizing, no?

Reply:
Yes, but as a method to reduce the diffs I blocked them for the time being to only report when latent sizing is requested. The thought is to open this up right after this big change is merged.